stable/stolon/templates/keeper-statefulset.yaml

```diff
@@ -0,0 +1,117 @@
+apiVersion: apps/v1beta2
+kind: StatefulSet
+metadata:
+  name: {{ template "stolon.fullname" . }}-keeper
+  labels:
+    app: stolon-keeper
```

Comment:
The `app` label should be `{{ template "stolon.name" . }}` throughout the chart.

Reply:
fixed.